cmd/create/create.go

```diff
@@ -0,0 +1,17 @@
+package create
+
+import (
+	"github.com/dream11/odin/cmd"
+	"github.com/spf13/cobra"
+)
+
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create resources",
```

Comment:
what other resources can be created through this apart from environments?

Reply:
label, service-set